spec/features/monthly_report_spec.rb

```diff
@@ -10,6 +10,23 @@
       before { visit monthly_reports_path }
       it { expect(first_report[:href]).to eq monthly_report_path(today) }
     end
+
+    describe 'search', js: true do
+      let(:url) { URI.parse(current_url) }
+
+      context 'by tag name' do
+        !let(:report) { create(:monthly_report, :with_tags, shipped_at: Date.today) }
+        let(:tags) { report.tags.name }
+
+        before do
+          visit monthly_reports_path("q[tags_name_in]" => tags)
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->